contents/handbook/growth/marketing/index.md

```diff
@@ -89,17 +89,24 @@ The Marketing team's main focus is acquisition of [high quality signups](https:/
-- [Product launches](https://posthog.com/handbook/growth/marketing/product-announcements). These bring transparency to the work that the PostHog team is doing. Product and engineering pull marketing here, not the other way around. Marketing's role is to look for cool things the team is already working on and to amplify them, often through a coordinated campaign that maximizes impact. Marketing's role is definitely _not_ to get teams to build features we think are 'marketable'. We define product broadly here - they can be anything from a major PostHog app feature to a cool internal tool that someone has built which we think the world should see. 
+  - _SEO content_ is high value and genuinely useful content that targets the keywords we think are most relevant to our customers. We do 'clean' SEO, ie. no buying backlinks or writing clickbait. 'No clickbait' doesn't mean content has to be boring though! Our aim is to increase our organic reach on Google, basically. We put monthly paid ads spend behind this type of content on LinkedIn or Twitter to increase awareness. 
+  - _Shareable content_ is stuff we think users and audiences outside PostHog will just find interesting. These may be technical deep dives or articles that give an insight into how PostHog works. We don't expect these to turn directly into signups, but they help us to build our unique and distinctive brand in the open source community. We share this content on 'super node' sites like Hacker News, Product Hunt and Reddit. 
+- [Sponsorship](/handbook/growth/marketing/open-source-sponsorship). We do this in two ways. 
+  - _Commercial sponsorship_ - Sponsoring newsletters, podcasts and events that are likely to reach our target audience. We try to keep a relatively narrow focus here. 
+  - _Open source sponsorship_ - we sponsor open source projects and developers that have helped us to build PostHog. Sometimes we do cross-promotion with them, but it's not necessary - this is more about giving back to the community. 
+- [Product launches](/handbook/growth/marketing/product-announcements). These bring transparency to the work that the PostHog team is doing. Product and engineering pull marketing here, not the other way around. Marketing's role is to look for cool things the team is already working on and to amplify them, often through a coordinated campaign that maximizes impact. Marketing's role is definitely _not_ to get teams to build features we think are 'marketable'. We define product broadly here - they can be anything from a major PostHog app feature to a cool internal tool that someone has built which we think the world should see. 
```

Comment:
```suggestion
- [Product launches](/handbook/growth/marketing/product-announcements). These bring transparency to the work we do. Product and engineering pull marketing here, not the other way around. Marketing's role is to look for cool things the team is already working on and to amplify them, often through a coordinated campaign that maximizes impact. Marketing's role is definitely _not_ to get teams to build features we think are 'marketable'. We define product broadly here - it can be anything from a major PostHog app feature to a cool internal tool that  we think the world should see. 
```